geode-wan/src/distributedTest/java/org/apache/geode/internal/cache/wan/parallel/ParallelWANPersistenceEnabledGatewaySenderDUnitTest.java

```diff
@@ -2231,6 +2240,102 @@ public void testpersistentWanGateway_restartSenderWithCleanQueuesDelayed_expectN
     vm3.invoke(() -> WANTestBase.validateRegionSize(getTestMethodName(), 0));
   }
 
+
+  /**
+   * Enable persistence for region as well as GatewaySender and see if remote site receives all the
+   * events.
+   */
```

Comment:
This comment doesn't seem to accurately reflect what this test is intending to show, since it doesn't mention possible duplicates at all. Would it be possible to get a better description of what this test is doing?

Reply:
Comment updated.